config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
+    bcrypt_password character(60),
+    CHECK (password IS NOT NULL OR bcrypt_password IS NOT NULL),
+    email character varying(254) NOT NULL,
+    joined timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
```

Comment:
We don't need to set this timestamp manually in the java code,
also I changed the type to timestamp with time zone, because in case the lobby server moves to a different time zone, the timestamps would still be correct.
More information here: https://stackoverflow.com/a/14070771

Reply:
Sounds okay. While on the topic, can we force all timestamps to be UTC somehow? I was looking at the DB rows the other day and had no clue which timezone, it looked like EST.

Reply:
It does not seem like we should even be doing this, can we force the timestamp to default to UTC? I really do not want any signficant timezones in our data. I'm picturing a case where a linode is migrated by linode from east to west coast, suddenly now we have to do timezone translation in queries (what a nightmare for what should have been very simple tasks like, "how many people joined in the last week"

Reply:
The DB should currently store all timestamps with the timezone that is configured to be used by postgres, by default the timezone the server is in.
I know it's possible to retrieve the values as UTC using SQL syntax, but I'm not sure if it works for insert statements as well.
Shouldn't matter too much as long as we don't end up with a completely wrong time at the end...

Reply:
It's more to maintain and wonder about.  WE should be using the `now()` function for this, so the timezone would always be server. Can we not do something to avoid thinking about timezones at all, force postgres to just use UTC always?

Reply:
On postgres NOW() is a synonym for CURRENT_TIMESTAMP
As I said, we definetely can always retrieve it as UTC value, we don't need to care about how it's stored inside the DB, postgres converts everything for us once I modify the queries.

Reply:
Sounds good, as long as there is no question about everything being UTC : )